docs/03 - Implementing the Anchor Server/Communication/Events Schema.yml

```
@@ -11,6 +11,7 @@ info:
 tags:
   - name: "Event Schemas"
     description: "Event Schemas for events that are published to a queue for the Anchor to consume."
+paths:
```

To solve the syntax error, I think you need this:
```suggestion
paths: {}
```

---

I do not think `status_change` is being used and not sure if it has a meaningful use case. The `status_change` should be derived from the log where we record every state of all fields.

The `source_account` and `destination_account` are available in [StellarTransactions.](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/docs/03%20-%20Implementing%20the%20Anchor%20Server/Communication/Platform%20API.yml#L355)

---

SGTM!

Do you think it's worth writing a small section in the README or CHANGELOG explaining how to proceed with this change? This way you can just ask our partners to look at that section to understand how to proceed with the updated version of the event.

---

Oh yeah... definitely. I will add CHANGELOG which is closer to the change.